### PR TITLE
Added the possibility to fetch userTypes based on a period

### PR DESCRIPTION
**Pull request by pactode**
It's easy to show the userTypes in a chart after fetching the results like this:
```php
$period = Period::days(7);
$userSessions = Analytics::fetchUserTypes($period);
```

### CODE REVIEW

File: src/Analytics.php [hunk 1]
```diff
@@ -116,6 +116,26 @@ public function fetchTopReferrers(Period $period, int $maxResults = 20): Collect
...
+    {
+        $response = $this->performQuery(
+            $period,
+            'ga:sessions',
+            [
+                'dimensions' => 'ga:userType',
+            ]
+        );
+
+        $userSessions = collect($response->rows ?? [])->map(function (array $usRow) {
```
Comment (freekmurze) [inline src/Analytics.php]: Let's return the collection directly (like in the other methods).

Use `userRow` instead of `usRow`

Comment (pactode) [reply src/Analytics.php]: Submitted the suggested changes.

**Comment by freekmurze** [comment]
Could you also add an example of this new function to the readme?

**Comment by pactode** [comment]
Updated the readme.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**